packages/analytics-js/src/services/StoreManager/storages/CookieStorage.ts

```diff
@@ -39,6 +40,8 @@ class CookieStorage implements IStorage {
     }
     this.isSupportAvailable = isStorageAvailable(COOKIE_STORAGE, this, this.logger);
     this.isEnabled = Boolean(this.options.enabled && this.isSupportAvailable);
+    delete this.options.enabled;
```

Comment:
Why do we have the `enabled` flag?

Reply:
The `enable` flag helps determine whether the storage type is enabled.